CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
```

Comment:
I've changed this to added. It doesn't look like anything was removed. Is that correct w.r.t the guidance on http://semver.org?